go.mod

```diff
@@ -146,7 +146,7 @@ require (
 	github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d // indirect
 	github.com/fatih/camelcase v1.0.0 // indirect
 	github.com/felixge/httpsnoop v1.0.3 // indirect
-	github.com/fxamacker/cbor/v2 v2.6.0 // indirect
+	github.com/fxamacker/cbor/v2 v2.7.0-beta // indirect
```

Comment:
Do we know when v2.7.0-beta will go GA?

Reply:
Originally there was to be a release candidate tag for `v2.7.0` over the past weekend. Several refactoring changes merged yesterday, so the maintainer is allowing extra fuzzer soak time before that happens (https://github.com/fxamacker/cbor/discussions/555). I've shared the KEP and 1.31 timelines with the maintainer and expect that we will be able to bump to a plain release tag before code freeze.